docs/FAQs.md

```diff
@@ -13,6 +13,109 @@ After you set up a repository, you can substitute in any tools you'd like.
 
 If you think the tool would be broadly useful to most consumers of this template, feel free to [file a feature request](https://github.com/JoshuaKGoldberg/create-typescript-app/issues/new?assignees=&labels=type%3A+feature&projects=&template=03-feature.yml&title=%F0%9F%9A%80+Feature%3A+%3Cshort+description+of+the+feature%3E) to add it in.
 
+## How can I create a GitHub action?
+
+Yes! If you want to read the [GitHub Actions documentation](https://docs.github.com/en/actions/creating-actions) in detail.
+Here we'll outline the steps required to migrate a CTA app to a GitHub Action:
+
+1. GitHub Actions store built output on a GitHub branch rather than in a published package on npm.
+   As a consequence we should:
+
+   - delete `.github/workflows/release.yml` and `.github/workflows/post-release.yml`.
+   - update `.github/workflows/build.yml` to ensure `dist` is up to date:
+
+     <details>
+         <summary><code>.github/workflows/build.yml</code></summary>
+
+     ```yml
+     jobs:
+       build:
+         runs-on: ubuntu-latest
+         steps:
+           - uses: actions/checkout@v4
+           - uses: ./.github/actions/prepare
+           - run: pnpm build
+
+           - name: Compare dist/index.js
+             run: |
+               if [ "$(git diff --ignore-space-at-eol --text dist/index.js | wc -l)" -gt "0" ]; then
+                 echo "Detected uncommitted changes after build."
+                 echo "You may need to run 'pnpm run build' locally and commit the changes."
+                 echo ""
+                 echo "See diff below:"
+                 echo ""
+                 git diff --ignore-space-at-eol --text dist/index.js
+                 echo ""
+                 # say this again in case the diff is long
+                 echo "You may need to run 'pnpm run build' locally and commit the changes."
+                 echo ""
+                 exit 1
+               fi
+
+           - id: upload
+             if: ${{ failure() && steps.diff.outcome == 'failure' }}
+             name: Upload artifact
+             uses: actions/upload-artifact@v4
+             with:
+               name: dist
+               path: dist/
```

Comment:
Similar to #1225: if we don't need an artifact, I don't think we should add config complexity to wrangle it.

```suggestion
```

In this case I think it should be enough to print the diff out and suggest running `pnpm run build`.